cli/index.js

```diff
@@ -39,13 +43,355 @@ const signUp = (args) => {
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not log out: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* recover 
+const recover = (args) => {
+
+    const { username, userId, email, newPassword, confirmPassword } = args;
+
+    const recoverUserData = {
+        username,
+        userId,
+        email,
+        newPassword,
+        confirmPassword,
+    };
+
+    console.log(chalk.yellow.bold("recovering..."));
+
+    axios
+        .put("/user/recover", recoverUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("recover successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not recover the account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* delete 
+const deleteuser = (args) => {
+
+    const { username } = args;
+
+    const deleteUserData = {
+        username,
+    };
+
+    console.log(chalk.yellow.bold("borrando..."));
+
+    axios
+        .delete("/user/{username}", deleteUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("the account has been deleted successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not delete this account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+// ==> user progile <===
+
+//* getuser 
+const getuser = (args) => {
+
+    console.log(chalk.yellow.bold("getting user..."));
+
+    const { username } = args;
+    
+        const getUserData = {
+            username
+        };
+
+    axios
+        .get("/user/{username}", getUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("you got the user successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not obtain the account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* updated user 
+const updateuser = (args) => {
+
+    const { userId, password, confirmPassword, username, firstName, 
+    lastName, email, bio, website, location, picture } = args;
+
+    const updateUserData = {
+        userId,
+        password,
+        confirmPassword,

```

Comment:
It doesn't make sense to update the password as part of the update call

Reply:
done

Reply:
again, cannot see it changed. Still has the same problems.